ui/apps/platform/src/constants/timeWindows.ts

```diff
@@ -15,9 +15,9 @@ export const snoozeDurations = {
 } as const;
 
 export const durations = {
-    HOUR: '1h',
-    DAY: '24h',
-    WEEK: '168h',
-    MONTH: '720h',
+    HOUR: '3600s',
+    DAY: '86400s', // 24 * 3600 seconds
+    WEEK: '604800s', // 7 * 24 * 3600 seconds
+    MONTH: '2592000s', // 30 * 24 * 3600 seconds
```

Comment:
It's JS there are libs for that :)
https://github.com/stdlib-js/constants-time-seconds-in-hour
https://github.com/stdlib-js/constants-time-seconds-in-day
https://github.com/stdlib-js/constants-time-seconds-in-week
I think an assumption that month has 30 days is not always true